charts/rasa-x/values.yaml

```diff
@@ -390,6 +390,11 @@ eventService:
 
 # app (custom action server) specific settings
 app:
+  # default is to install action server from image.
+  install: true
+  # if install is set to false, the url to the existing action server can be configured by setting existingURL.
```

Comment:
Just seen, that there is an upper-lowercase typo in this line. It should be "existingUrl" not "...URL".